### DISCUSSION TITLE
Support for APICurio?

### DISCUSSION THREAD

**Discussion by robindenadel** [sngular/kloadgen#300] [Ideas]
We are trying to use kloadgen to test an application using APICurio for ser/des and schema registry.
Doing that we ran into problems because of APICurio/confluent compatibility issues.
When using the confuent KafkaAvroSerializer we get an error like described in https://github.com/Apicurio/apicurio-registry/issues/2547.

It would be great if we could use io.apicurio.registry.serde.avro.AvroKafkaSerializer as the serializer. Are there any plans on supporting APICurio?

**Comment by jemacineiras** [comment]
Hi @robindenadel,

Nowadays, our roadmap is almost empty, so your proposal gives us new air. We will analyse this registry and create the necessary task to support it.

Cheers!!
